### PR TITLE
Stop using PrettyXML

### PR DESCRIPTION
**Pull request by walro**
The transform is taken from the PrettyXML gem.

### CODE REVIEW

File: feedjira.rb [hunk 1]
```diff
@@ -33,7 +33,12 @@
     end
 
     xml = connection.get(params[:url]).body
-    prettyxml = PrettyXML.write(xml)
+    parsed_xml = Nokogiri::XML(xml)
+
+    xslt_file = File.open("./files/pretty_xml.xslt", "r")
```
Comment (jage) [inline feedjira.rb]: Nitpick, but how about `File.join(".", "files", "pretty_xml.xslt")`?

**Comment by dentarg** [comment]
What about putting the XSLT stuff in a class (maybe call it PrettyXML? :))? I like abstractions

Also, I guess we need to do include the MIT license here now as https://github.com/jrobertson/pretty-xml uses that

**Comment by walro** [comment]
> What about putting the XSLT stuff in a class (maybe call it PrettyXML? :))? I like abstractions

Hehe in general I'd agree, but nothing else is abstracted here so I think we can live with quick-and-dirty here? :)

> Also, I guess we need to do include the MIT license here now as https://github.com/jrobertson/pretty-xml uses that

Hmm, I wonder what the source of the xslt is. For example, the exact same one can be found here: https://github.com/mislav/gorgeous/blob/bd91d3d0ad2ef1e1072853f7249cc4aa7521de86/lib/pretty.xslt and http://stackoverflow.com/questions/10219930/strange-xml-formatting-with-xslt-stylesheet

But we can MIT license anyway, suppose it won't hurt.

**Comment by dentarg** [comment]
> Hehe in general I'd agree, but nothing else is abstracted here so I think we can live with quick-and-dirty here? :)

Yeah, I know but it was abstracted before! :)

**Comment by walro** [comment]
> Yeah, I know but it was abstracted before! :)

hehe yeah, that particular thing. but I don't see a http client etc! :smile:

**Comment by dentarg** [comment]
It's okay, LGTM

File: feedjira.rb (hunk 1 above)
Comment (walro) [inline feedjira.rb]: Ah yeah. Thanks!

**Comment by dentarg** [comment]
✨

**Pull request merged**